Metempsychotic machine that definitely works.

PHCodes opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.

Check the box below to confirm that you have in fact seen this (put an X in the brackets, like [X]):
-->

- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->
Makes my body scanners break on every server instance I run. Hoping to fix this by restarting. 
**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->

<!--
Make sure to take this Changelog template out of the comment block in order for it to show up.
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->

toasterpm87 commented:
shit?

toasterpm87 commented:
i pull it up to nayno once magil is back or something

toasterpm87 commented:
i will look into the error on friday

FluffiestFloof commented on Resources/Prototypes/DeltaV/Entities/Objects/fun/zero.yml:
```diff
@@ -0,0 +1,15 @@
+ type: entity
```
You forgot a dash here.

toasterpm87 replied:
thanks

PHCodes committed:
Metempsychotic machine that definitely works.

PHCodes committed:
Merge branch 'master' into metempsychosis

PHCodes committed:
Cleans up some stuff and gets the system into a demoable state.

Someone else needs to test this, the physics appear to be broken in my local machine.

PHCodes committed:
Merge branch 'master' into metempsychosis

PHCodes committed:
Merge branch 'metempsychosis' of https://github.com/PHCodes/Delta-v-rebase into metempsychosis

PHCodes committed:
Merge branch 'master' into metempsychosis

Signed-off-by: JJ <47927305+PHCodes@users.noreply.github.com>

DebugOk commented:
The scanners have been fixed, if you resolve the merge conflicts and make sure this works I can go test it
